src/components/student-dashboard/all-events-by-week.cjsx

```diff
@@ -0,0 +1,28 @@
+React  = require 'react'
+BS     = require 'react-bootstrap'
+moment = require 'moment'
+EventsPanel = require './events-panel'
+{StudentDashboardStore} = require '../../flux/student-dashboard'
+_ = require 'underscore'
+
+module.exports = React.createClass
+  displayName: 'AllEventsByWeek'
+  propTypes:
+    courseId: React.PropTypes.any.isRequired
+
+  renderWeek:(events, week) ->
+    startAt = moment(week, "YYYYww")
+    <EventsPanel
+      key={week}
+      courseId={@props.courseId}
+      events={events}
+      startAt={startAt}
+      endAt={startAt.clone().add(1, 'week')}
```

Comment:
Yeah it is.  A surprising thing about the `moment` lib is that all methods simply mutate the object vs returning a new one.  So the `.add(1, 'week')` would actually modify the startAt var, causing the earlier set of the `startAt` prop to be identical to `endAt`.  The call to `clone()` causes a new object to be created so it's not shared.  There's several other spots in the code that have the same pattern.  http://momentjs.com/docs/#/parsing/moment-clone/